pkg/controller/deadmanssnitch/deadmanssnitch_controller.go

```diff
@@ -125,6 +126,11 @@ func (r *ReconcileDeadMansSnitch) Reconcile(request reconcile.Request) (reconcil
 	reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
 	reqLogger.Info("Reconciling DeadMansSnitch")
 
+	start := time.Now()
+	defer func() {
+		localmetrics.Collector.ObserveReconcile(time.Since(start).Seconds())
```

Comment:
Other impls put a "Reconcile complete" log message in here that includes the duration. Probably not a must-have, but could be nice.